aws-msk/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: template-quickstart
+
+# Displayed in the UI (required)
+title: Template Quickstart
```

Comment:
This appears to also be the template placeholder values, please fill this file out with information relevant to your quickstart!